OsmAnd/src/net/osmand/plus/OsmandSettings.java

```diff
@@ -252,6 +254,18 @@ public void migrateGlobalPrefsToProfile() {
 		}
 	}
 
+	public void migrateHomeWorkParkingToFavorites() {
+		FavouritesDbHelper favorites = ctx.getFavorites();
+		favorites.addFavoritePersonal(getHomePoint(),FavouritesDbHelper.HOME,getHomePointDescription().getName());
+		favorites.addFavoritePersonal(getWorkPoint(),FavouritesDbHelper.WORK,getWorkPointDescription().getName());
+		if (OsmandPlugin.getEnabledPlugin(ParkingPositionPlugin.class) != null) {
```

Comment:
Should not be here, no plugin class in settings. Either parking plugin should migrate by itself or the point could be lost after upgrade